### PR TITLE
Hide faxes section when unavailable

### PR DESCRIPTION
**Pull request by danigargar**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

File: web/portal/user/src/App.tsx [hunk 1]
```diff
@@ -24,6 +24,18 @@ export default function App(): JSX.Element {
...
+    (actions) => actions.userStatus.status.resetProfile
+  );
+
+  const userStatusInit = useStoreActions(
+    (actions) => actions.userStatus.status.init
+  );
+
+  const loadStatus = useStoreActions(
+    (actions) => actions.userStatus.status.load
+  );
```
Comment (mmadariaga) [inline web/portal/user/src/App.tsx:37]:
```suggestion
  const { resetProfile: userStatusResetProfile, init: userStatusInit, load: loadStatus } = useStoreActions(
    (actions) => actions.userStatus.status
  );
```
Or
```suggestion
  const status = useStoreActions(
    (actions) => actions.userStatus.status
  );
  const userStatusResetProfile = status.resetProfile;
  const userStatusInit = status.init;
  ...
```

File: web/portal/user/src/App.tsx [hunk 2]
```diff
@@ -36,7 +48,20 @@ export default function App(): JSX.Element {
...
+    loadStatus();
+  }, [
+    apiSpecStore,
+    authStore,
+    token,
+    setLanguages,
+    setLoginProps,
+    userStatusResetProfile,
+    userStatusInit,
+    loadStatus,
```
Comment (mmadariaga) [inline web/portal/user/src/App.tsx:63]: I don't get why you need these new effect triggers

File: web/portal/user/src/store/userStatus/status.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  // actions
+  setProfile: action<StatusStore, Status>((state, profile) => {
+    state.profile = profile;
+  }),
+  resetProfile: action<StatusStore>((state) => {
+    localStorage.removeItem('profile');
+    state.profile = null;
+  }),
+  init: thunk<StatusStore>(async (actions) => {
+    const profile = localStorage.getItem('profile') as string | undefined;
```
Comment (mmadariaga) [inline web/portal/user/src/store/userStatus/status.ts:58]: I don't think we need to store this at all

File: web/portal/user/src/router/useEntityMap.ts [hunk 4]
```diff
@@ -1,9 +1,22 @@
...
+  const [entityMap, setEntityMap] = useState<ExtendedRouteMap>(getEntityMap());
+
+  useEffect(() => {
+    setEntityMap(getEntityMap());
+  }, [status]);
+
+  const aclFilteredEntityMap = useFeatureFilteredEntityMap({
+    entityMap,
+    status,
+  });
```
Comment (mmadariaga) [inline web/portal/user/src/router/useEntityMap.ts:19]: We could optionally optimize by calculating entityMap just once. Something like
```suggestion
  const [entityMap, setEntityMap] = useState<ExtendedRouteMap>([]);

  useEffect(() => {
    if (entityMap.length > 0) {
      return;
    }
    setEntityMap(getEntityMap());
  }, [status]);

  const aclFilteredEntityMap = useFeatureFilteredEntityMap({
    entityMap,
    status,
  });
```

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/user/src/store/userStatus/status.ts (hunk 3 above)
Comment (danigargar) [reply web/portal/user/src/store/userStatus/status.ts:58]: It doesn't work without it.

File: web/portal/user/src/App.tsx (hunk 2 above)
Comment (danigargar) [reply web/portal/user/src/App.tsx]: The linter complaints if those are missing:
`React Hook useEffect has missing dependencies: 'loadStatus' and 'userStatusInit'. Either include them or remove the dependency array`

File: web/portal/user/src/router/useEntityMap.ts (hunk 4 above)
Comment (danigargar) [reply web/portal/user/src/router/useEntityMap.ts:19]: I have done it but the component still gets reloaded.

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**